gdk-pixbuf/src/pixbuf.rs

```diff
@@ -35,7 +35,7 @@ impl Pixbuf {
         let bits_per_sample = bits_per_sample as usize;
 
         let n_channels = if has_alpha { 4 } else { 3 };
-        let last_row_len = width * ((n_channels * bits_per_sample + 7) / 8);
+        let last_row_len = width * (n_channels * bits_per_sample).div_ceil(8);
```

Comment:
This is missing a 7 isn't it?

Reply:
No, why?

Reply:
There was a +7 before and that change seemed unrelated to clippy fixes.

Reply:
That `(x + 7) / 8` is the same as dividing by 8 and rounding up (which is what `div_ceil()` does)